openapi3.yaml

```diff
@@ -61,12 +147,37 @@ components:
       properties:
         message:
           type: string
-    resource:
+    requestBodyForFilter:
+      type: object
+      required:
+        - arrayOfilterProduct
+      properties:
+        arrayOfilterProduct:
+          type: array
+          items:
+            $ref: '#/components/schemas/filterProduct'
+        limit:
+          type: integer
+          minimum: 0
+        skip:
+          type: integer
+          minimum: 0
+    filterProduct:
+              # expmple: {feild:'minZoom',oprator:'>',filterArg:'10'}
       type: object
       required:
-        - id
-        - name
-        - description
+        - feild
+        - oprator
+        - filterArg
+      properties:
+        feild:
+          type: string
+        operator:
```

Comment:
Why is this `string`? You know the supported operation.

Reply:
ok got it. i changed that for:
```
      properties:
        feild:
          type: string
          enum:
            - id
            - name
            - description
            - bounding_polygon
            - consumtion_link
            - type
            - consumption_protocol
            - resolution_best
            - min_zoom
            - max_zoom
        operator:
```
I didnt use camelCase on purpose because the columns in the postgres DB cant be on camelCase.